apps/web/components/v2/eventtype/AvailabilityTab.tsx

```diff
@@ -0,0 +1,146 @@
+
+import { AvailabilitySelectSkeletonLoader } from "@components/availability/SkeletonLoader";
+
+type AvailabilityOption = {
+  label: string;
+  value: number;
+};
+
+const AvailabilitySelect = ({
+  className = "",
+  ...props
+}: {
+  className?: string;
+  name: string;
+  value: number;
+  onBlur: () => void;
+  onChange: (value: AvailabilityOption | null) => void;
+}) => {
+  const query = trpc.useQuery(["viewer.availability.list"]);
+
+  return (
+    <QueryCell
+      query={query}
+      customLoader={<AvailabilitySelectSkeletonLoader />}
+      success={({ data }) => {
+        const options = data.schedules.map((schedule) => ({
+          value: schedule.id,
+          label: schedule.name,
+        }));
+
+        const value = options.find((option) =>
+          props.value
+            ? option.value === props.value
+            : option.value === data.schedules.find((schedule) => schedule.isDefault)?.id
+        );
+        return (
+          <Select
+            options={options}
+            isSearchable={false}
+            onChange={props.onChange}
+            className={classNames("block w-full min-w-0 flex-1 rounded-sm text-sm", className)}
+            value={value}
+          />
+        );
+      }}
+    />
+  );
+};
+
+const format = (date: Date) =>
+  Intl.DateTimeFormat(undefined, { hour: "numeric", minute: "numeric" }).format(
+    new Date(dayjs.utc(date).format("YYYY-MM-DDTHH:mm:ss"))
+  );
+
+export const AvailabilityTab = () => {
+  const { t, i18n } = useLocale();
+  const { watch } = useFormContext<FormValues>();
+
+  const scheduleId = watch("schedule");
+  const { isLoading, data: schedule } = trpc.useQuery(["viewer.availability.schedule", { scheduleId }]);
+
+  const filterDays = (dayNum: number) =>
+    schedule?.schedule.availability.filter((item) => item.days.includes((dayNum + 1) % 7)) || [];
+
+  return (
+    <>
+      <div>
+        <div className="min-w-4 mb-2">
+          <label htmlFor="availability" className="mt-0 flex text-sm font-medium text-neutral-700">
+            {t("availability")}
+          </label>
+        </div>
+        <Controller
+          name="schedule"
+          render={({ field }) => (
+            <AvailabilitySelect
+              value={field.value}
+              onBlur={field.onBlur}
+              name={field.name}
+              onChange={(selected) => {
+                field.onChange(selected?.value || null);
+              }}
+            />
+          )}
+        />
+      </div>
+
+      <div className="space-y-4 rounded border p-8 py-6 pt-2">
+        <ol className="table border-collapse text-sm">
+          {weekdayNames(i18n.language, 1, "long").map((day, index) => {
+            const isAvailable = !!filterDays(index).length;
+            return (
+              <li key={day} className="my-6 flex border-transparent last:mb-2">
+                <span className={classNames("w-32 font-medium", !isAvailable && "text-gray-500")}>{day}</span>
```

Comment:
```suggestion
                <span className={classNames("w-32 font-medium", !isAvailable && "text-gray-500 opacity-50")}>{day}</span>
```

Reply:
I was wondering why in Figma it didn't have the same colour

Reply:
Yeah a few places - especially disabled states are just modified with opacity